src/main/java/com/ksm/wstbackoffice/controller/AddressController.java

```diff
@@ -30,7 +30,14 @@ public AddressController(AddressService addressService) {
 
     @GetMapping("{id}")
     public ResponseEntity<Address> findById(@PathVariable long id) {
+        if (id < 0 || id == 0)
```

Comment:
1. Try never to use if statements without square brackets. Even though java language allows us to do it. Because the code sometimes can be difficult to understand.
2. Can you change you check in order to have the only one comparison instead of doing both I'd < 0 and I'd == 0? It's just going to be faster

Reply:
done